.github/workflows/automerge.js

```diff
@@ -0,0 +1,137 @@
+let branch = '';
+let actor = '';
+let prName = '';
+
+if (process.argv.length >= 3) { branch = process.argv[2]; }
+if (process.argv.length >= 4) { prName = process.argv[3]; }
+if (process.argv.length >= 5) { actor = process.argv[4]; }
+
+console.log('--------------------------------------------------------');
+
+if (!branch)
+{
+    console.log('Branch name argument (first) was not specified');
+    return;
+}
+
+if (!prName)
+{
+    console.log('prName argument (second) was not specified');
+    return;
+}
+
+if (!actor)
+{
+    console.log('Actor argument (third) was not specified');
+    return;
+}
+
+console.log(branch);
+console.log(prName);
+console.log(actor);
+
+if (actor != 'dependabot')
+{
+    console.log('Automerge works only for PRs created by dependabot.');
+    return;
+}
+
+if (prName.includes('patternfly'))
+{
+    console.log('Automerge cant merge patternfly Prs.');
+    return;
+}
+
+if (prName.includes('types/node'))
+{
+    console.log('Checking for types node version.');
+    let pattern = /from 16[.]\d\d[.]\d\d to 16[.]\d\d[.]\d\d/;
+    let res = pattern.test(prName);
+    if (res)
+    {
+        console.log('Version does match the pattern ' + pattern);
+    }else
+    {
+        console.log('Version does not match the pattern ' + pattern);
+        return;
+    }
+
+
+}
+
+const { exec } = require("child_process");
```

Comment:
`child_process` is a node api, not an npm package

so, this works, but the new convention (since node16) is 
```suggestion
const { exec } = require("node:child_process");
```